kctf-operator/pkg/resources/daemon-ctf.go

```diff
@@ -0,0 +1,91 @@
+package resources
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func NewDaemonSetCtf() runtime.Object {
```

Comment:
Not that it's planned, but if we wanted to ship some binary files with the daemon set in the future, any idea how we could do this?

Reply:
Maybe we can provide these daemons as container images on gcr.io or so in the future

Reply:
What do you mean by shipping some binary files? And how could we do it with yaml files?

Reply:
ah, I was thinking mainly that we might want to replace that shell command with a binary at some point. But yeah, for that we would have to build a docker image anyway.
On the command line, you can do something like `kubectl create configmap --from-file=` but a docker image would be better in that case anyway I think. So nevermind, doesn't sound like something we need to think about right now.